futures-util/src/future/future/map.rs

```diff
@@ -1,12 +1,12 @@
 use core::pin::Pin;
 use futures_core::future::{FusedFuture, Future};
 use futures_core::task::{Context, Poll};
-use pin_project::{pin_project, project};
+use pin_project::pin_project;
 
 use crate::fns::FnOnce1;
 
 /// Internal Map future
-#[pin_project(Replace)]
+#[pin_project(Replace, project = MapProj, project_replace = MapProjOwn)]
```

Comment:
It is not ideal that we need to have both `Replace` and `project_replace = <name>` as `project_replace = <name>` implies that a `project_replace` method is generated. 
I'll fix it soon.

Reply:
Fixed in pin-project 0.4.20